docs/human-interface-guidelines.md

```diff
@@ -657,153 +657,188 @@ A Dynamic Notebook is a way for an app to provide user-managable tabbing functio
 
-#### Highlights {#highlights}
+Action icons are used to represent common user actions, such as "delete", "play" or "save". These icons are mostly found in app toolbars, but can be found throughout the OS.
 
-The edges of objects tend to reflect light more due to the fact the position of the observer relative to the light source is almost always ideal for the reflection. We produce a subtle second inner outline of the object as a highlight. This stroke is very subtle and may not be apparent on some matte objects.
+<div style="width:100%;display:inline-block;text-align:center;">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/go-previous.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/go-next.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/document-export.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/document-print.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/document-save-as.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/edit-delete.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/edit-cut.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/edit-undo.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/object-inverse.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/media-playback-start.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/tag-new.svg">
+	<img class="hig-icon small" src="/images/docs/human-interface-guidelines/icons/24/open-menu.svg">
+</div>
 
-#### Limited glossy reflections {#limited-glossy-reflections}
+If your app makes use of a one of these common actions, reference its corresponding icon instead of creating your own. This ensures a consistent user experience and aids in user recognition of common functions.
 
-Use glossy reflection only on objects that have a reflective surface in real life (plastic, glass, some metal, et cetera). A sheet of paper should not be glossy.
+However if your app has a unique action, you may need to create your own. When doing this, try to follow the look and feel of existing action icons and include it along with your app.
 
-### Lighting and Perspective {#lighting-and-perspective}
 
-elementary Icons have a consistent light source and perspective to keep a similar look throughout the system. The icons have been designed with the UI theme in mind, meaning everything shares a cohesive feel and a subtle amount of realism.
+## Icon Sizes {#size}
 
-#### Lighting {#lighting}
+elementary OS uses **six** main icon sizes throughout the desktop. So if you're developing an application for elementary or making an icon set, it's best to include all six.
```

Comment:
> So if you're developing an application for elementary...

That sentence puts me on edge. _"So if you're..."_ sounds too friendly, and this is the HIG, who would be here except developers?